Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAutomationWorker.py

```diff
@@ -201,6 +224,12 @@ def Get_Marshall(WorkspaceId, Enabled, AzureDnsAgentSvcZone):
 KEYRING_PATH="/etc/opt/omi/conf/omsconfig/keyring.gpg"
 LOCAL_LOG_LOCATION = "/var/opt/microsoft/omsagent/log/nxOMSAutomationWorker.log"
 
+# permisson level rwx rwx ---
+# leading zero is necessary because this is an octal number
+FILE_PERMISSION_LEVEL = 0770
+
+AUTOMATION_USER = "nxautomation"
+
 def is_oms_primary_workspace(mof_workspace_id):
```

Comment:
Minor : I would have expected "workspace_id" as a parameter not "mof_..." This method could be used with any id, why name the parameter explicitly for mof?